src/app/views/query-response/graph-toolkit/GraphToolkit.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { connect } from 'react-redux';
+
+import { Label } from 'office-ui-fabric-react';
+import { IQuery } from '../../../../types/query-runner';
+import { parseSampleUrl } from '../../../utils/sample-url-generation';
+
+const templateMap: any = {
+  '/me/planner/tasks' : 'https://mgt.dev/iframe.html?id=components-mgt-tasks--tasks',
+  '/me': 'https://mgt.dev/iframe.html?id=components-mgt-person-card--person-card-hover',
+  '/me/people' : 'https://mgt.dev/iframe.html?id=components-mgt-people--people',
+};
+
+class GraphToolkit extends Component<any> {
+  constructor(props: any) {
+    super(props);
+  }
+
+  public lookupToolkitUrl = (sampleQuery: IQuery): any => {
+    if (sampleQuery) {
+      const { requestUrl, search } = parseSampleUrl(sampleQuery.sampleUrl);
+      const query = requestUrl + search;
+      // find if the url of the request has a template mapped to it
```

Comment:
With the templates stored in an object/dictionary data structure, we can get the value in constant time. Ie 
```javascript
templateMap[query]
```